app/services/bulk_segmented_users.rb

```diff
@@ -0,0 +1,51 @@
+class BulkSegmentedUsers
```

Comment:
I bring this up reluctantly, but we have [a few conventions we try to follow with service classes](https://developers.forem.com/backend/service-objects#what-are-service-objects) — `app/services/{things}/do_them`, VerbNoun-naming, `call`.

Maybe these could be `SegmentedUsers::Upsert` and `SegmentedUsers::Delete` ?

Reply:
Oh that's fair, and I don't think you should be reluctant about it! Ridhwana mentioned during onboarding but I forgot 😅